src/webui/www/private/views/preferences.html

```diff
@@ -985,6 +985,17 @@
                     </select>
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="torrentContentRemoveOption">QBT_TR(Torrent content removing mode:)QBT_TR[CONTEXT=OptionsDialog]</label>
+                </td>
+                <td>
+                    <select id="torrentContentRemoveOption" style="width: 15em;">
+                        <option value="Delete">QBT_TR(Delete files permanently)QBT_TR[CONTEXT=OptionsDialog]</option>
+                        <option value="MoveToTrash">QBT_TR(Move files to Trash)QBT_TR[CONTEXT=OptionsDialog]</option>
```

Comment:
Why the capital letter on `trash`? Also shouldn't it rather be `Move files to trash if possible` because its not guaranteed that files can be moved to trash (on Windows). For example if they're on a network drive. Some people might not know and think its a secure solution.

Reply:
> Why the capital letter on `trash`? Also shouldn't it rather be `Move files to trash if possible` because its not guaranteed that files can be moved to trash (on Windows). For example if they're on a network drive. Some people might not know and think its a secure solution.

Changed.